Dockerfile.centos

```diff
@@ -79,6 +81,9 @@ RUN wget -qO- https://github.com/ccache/ccache/releases/download/v3.7.7/ccache-3
   && cd ../ && rm -rf ccache-*
 
 # Install lcov to /usr/local/bin, require master to make gcc9 happy
-RUN git clone --depth=1 --branch=master --single-branch https://github.com/linux-test-project/lcov.git \
-  && cd lcov && git checkout 40580cd65909bc8324ae09b36bca2e178652ff3f \
+RUN git clone --branch=master --single-branch https://github.com/linux-test-project/lcov.git \
+  && cd lcov && git checkout 75fbae1cfc5027f818a0bb865bf6f96fab3202da \
   && make install && cd ../ && rm -rf lcov
+
+# fastcov
+RUN pip3 install --no-cache-dir fastcov
```

Comment:
This individual open-source has not been very widely used, it should be more verification？

Reply:
Yes, I had the same problem before. But seems no better way to speed up this now.